code/src/components/TodoList.js

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react'
+import { useSelector } from 'react-redux'
+import styled from 'styled-components'
+
+import { Todo } from 'components/Todo'
+import { Select } from 'library/Select'
+import { ClearButton } from 'components/ClearButton'
+import logo from '../assets/bored-cat.gif'
+// It is a bit confusing that I use both items and todos below (need to look at this when I have time).
+export const TodoList = () => {
+  const [category, setCategory] = useState('')
+  const items = useSelector((store) => store.todos.items)
+
+  const todos = useSelector((store) => {
+    if (!category) return store.todos.items //if no category, show all items in list
+    else return store.todos.items.filter(item => item.category === category) // else filter the list on category choosen
+  })
+  
+  if (items.length === 0) {
+    return (
+      <ListWrapper>
+        <CatGif src={logo} alt='bored cat' />
```

Comment:
I like this event! Really nice choice of cat :D

Reply:
Glad you liked the cat :)